2.0/m_blockhighlight.cpp

```diff
@@ -74,6 +77,14 @@ class ModuleBlockHighlight : public Module
 		if (chan->GetUsers()->size() < minusers)
 			return MOD_RES_PASSTHRU;
 
+		// We only work if the channel mode is enabled.
+		if (!chan->IsModeSet(mode.GetModeChar()))
+			return MOD_RES_PASSTHRU;
```

Comment:
Is this the most efficient order? I think we should check if the mode is active before we check the channel size and the message length.